src/library_pthread.js

```diff
@@ -693,7 +693,7 @@ var LibraryPThread = {
     if (transferredCanvasNames) transferredCanvasNames = UTF8ToString(transferredCanvasNames).trim();
     if (transferredCanvasNames) transferredCanvasNames = transferredCanvasNames.split(',');
 #if GL_DEBUG
-    console.log('pthread_create: transferredCanvasNames="' + transferredCanvasNames + '"');
+    err('pthread_create: transferredCanvasNames="' + transferredCanvasNames + '"');
```

Comment:
```suggestion
    out('pthread_create: transferredCanvasNames="' + transferredCanvasNames + '"');
```

Reply:
The GL_DEBUG elsewhere in this file (and in other files) seems to prefer `err()`.. so I'm going with consistency with that.   I don't know why it does that..   Maybe I'll change the other one in this file..